### PR TITLE
Added downloading to Hyperpartisan news detection

### PR DESCRIPTION
**Pull request by ghomasHudson**
Following the discussion on Slack and #349, I've updated the hyperpartisan dataset to pull directly from Zenodo rather than manual install, which should make this dataset much more accessible. Many thanks to @johanneskiesel !

Currently doesn't pass `test_load_real_dataset` - I'm using `self.config.name` which is `default` in this test. Might be related to #474

### CODE REVIEW

**Comment by johanneskiesel** [comment]
Thank you @ghomasHudson for making our dataset available! This is great!

**Comment by lhoestq** [comment]
The tests passes since #527 :)

File: datasets/hyperpartisan_news_detection/hyperpartisan_news_detection.py [hunk 1]
```diff
@@ -111,42 +100,44 @@ def _info(self):
...
-                    gen_kwargs={
-                        "articles_file": os.path.join(
-                            data_dir, "articles-validation-" + self.config.name + "-20181122.xml"
-                        ),
-                        "labels_file": os.path.join(
-                            data_dir, "ground-truth-validation-" + self.config.name + "-20181122.xml"
-                        ),
-                    },
+                    name=split,
+                    gen_kwargs = data_dir[split]
```
Comment (lhoestq) [inline datasets/hyperpartisan_news_detection/hyperpartisan_news_detection.py:132]: Could you run `make style` before we merge ?

**Review by lhoestq — APPROVED** [review]
Thanks for updating the link :)

**Pull request merged**